### PR TITLE
Minor changes which I think makes crass nicer

### PR DESCRIPTION
**Pull request by stoivo**
For the last couple of days I have been looking at crass and how it parses CSS. I didn't end up making any big changes. I did some changed that I think is nice. I wonder if you are interested in merging any of these changed?

### CODE REVIEW

File: test/css-parsing-tests/README.rst
Comment (rgrove) [inline test/css-parsing-tests/README.rst:1]: This file is part of a git subtree and shouldn't be changed here. The upstream repo for these tests is at https://github.com/CourtBouillon/css-parsing-tests

**Review by rgrove — CHANGES_REQUESTED** [review]
Thanks for the contributions.

I appreciate the spec URL updates in commit ff4b3e4 (minus the changes to `test/css-parsing-tests/README.rst`, which is in a git subtree and shouldn't be modified) and the test name improvement in commit 08f6145.

However, I'm not interested in the refactoring in commits 91fe420 and 6a71dd7. Those are semver-major changes that would break backwards compatibility, and I don't think they're necessary.

**Comment by stoivo** [comment]
Updated the PR to only two commit you wanted

> However, I'm not interested in the refactoring in commits https://github.com/rgrove/crass/commit/91fe4208829f5766c9e152bcb63090ad750b457d and https://github.com/rgrove/crass/commit/6a71dd784e12c7874d7947f2d320f44270c6f1df. Those are semver-major changes that would break backwards compatibility, and I don't think they're necessary. 

If I could make it so it isn't breaking semver would you like to include it then? 6a71dd784e12c7874d7947f2d320f44270c6f1df it easy to do without breaking semver. 91fe4208829f5766c9e152bcb63090ad750b457d can't be done without breaking semver but I don't think these methods was intendent to be public

**Review by rgrove — APPROVED** [review]
> Updated the PR to only two commit you wanted

Thank you!

> If I could make it so it isn't breaking semver would you like to include it then? https://github.com/rgrove/crass/commit/6a71dd784e12c7874d7947f2d320f44270c6f1df it easy to do without breaking semver. https://github.com/rgrove/crass/commit/91fe4208829f5766c9e152bcb63090ad750b457d can't be done without breaking semver but I don't think these methods was intendent to be public

After giving this some thought, I'd prefer not to reorganize things this way, at least for now.

`Crass::Parser` is intended to be public, even though most users shouldn't need to use it. The methods on `Crass` are aliases that provide the "simple" API that most users will use, whereas `Crass::Parser` contains the actual parser implementation and exposes lower level functionality that most users won't need unless they're implementing their own custom parsers.

That said, I would be open to making `Crass::stringify` an alias for `Crass::Parser.stringify`, but only an alias — I don't want to move the actual implementation to `Crass`.

**Comment by stoivo** [comment]
Do you think users are calling instance method `consume_component_value` and `consume_declaration`, and this is public api?

**Pull request merged**